admin/controllers/download-file.js

```diff
@@ -0,0 +1,8 @@
+'use strict'
+
+module.exports = async (req, res) => {
+  const fileName = 'MTC_CAG_Feb_2018_Trial.pdf'
+  res.setHeader('Content-disposition', 'filename=' + fileName)
+  res.setHeader('Content-type', 'application/pdf')
+  res.download('public/PDFs/' + fileName)
```

Comment:
If the path `public/PDFs/` is actually public then the file can still be downloaded even if the user is not logged in, although guessing the filename is unlikely.

Is this something we want to prevent?

Reply:
come to think of it - have we tried fixing this with something as simple as `target=_blank` to force it into a new window?

Reply:
@jon-shipley yes, we need to protect 'public' folder too, and I'm also planning on moving those unique file names to either config or a specific file config file. This is tech debt that I'm hoping I'll be able to address after the feb trial.

Reply:
@pris54 - protecting the public folder sounds unusual - normally we just serve the file from outside the web root.

Is that a part of this PR?

Reply:
@jon-shipley not part of this PR. I find unusual that we need to protect files that are effectively public as they are part of this repository =) I believe the logic is that they are public to certain roles. More clarification will be needed (soon) before addressing this particular tech debt (renaming the 'public' folder? removing it completely and server from 'assets'?)

Reply:
@jon-shipley that may be also a good opportunity to look more deeply into role management.

Reply:
sounds like our `public` folder is more of a `static` folder.
the only question i have is whether we *really* need to serve this through express.  
Its in the github repo, so is effectively public anyway.
Has anyone indicated it should be accessible to authorised users only?

Reply:
@GuyHarwood those questions are pretty much the ones I've just posted in reply to Jon's comments. I think more clarification is needed after the feb trial, I'm making a note to discuss with Paul S. and BAs in the next planning meeting.

Reply:
Oh yes. Good point 🤦‍♂️ 

In that case I question why we even want node to serve the files at all. Better solutions exist.  I'm happy to approve this as-is, as there is no other architecture in place at this time.

Reply:
@jon-shipley Agreed. Please see the bottom note on my previous reply to Guy.